scripts/EN/Harry

```diff
@@ -5664,24 +5664,24 @@ LoadBackground(0x0); // 0x7248: 00001069
+ShowText((1590)"Odd.¥nW2I do not see the brave attitude that was present in her just before. ¥nW2She walks as if something has drained the life out of her, W1like she is wandering about for no reason. W2I approach her.¥nW2And call out to her once more."); // 0x725c: 0000100c
 HandleInput(); // 0x7262: 00000010
 LoadSecondEffect(0x39, 0x1, 0x1); // 0x7266: 0000306e
-ShowText((1591)"As I do this Cybil staggers as she slowly turns, and confirms my presence."); // 0x7270: 0000100c
+ShowText((1591)"As I do this Cybil staggers as she slowly turns, W1and confirms my presence."); // 0x7270: 0000100c
 HandleInput(); // 0x7276: 00000010
-ShowText((1592)"And then, as if I am watching her in slow motion she raises her right arm until it is parallel with the ground and then a bullet fires from the handgun she is holding."); // 0x727a: 0000100c
+ShowText((1592)"And then, W1as if I am watching her in slow motion W1she raises her right arm until it is parallel with the ground W1and then a bullet fires from the handgun she is holding."); // 0x727a: 0000100c
 HandleInput(); // 0x7280: 00000010
 LoadSecondEffect(0x39, 0x2, 0x1); // 0x7284: 0000306e
 PlaySecondSoundFx(0xf); // 0x728e: 00001073
-ShowText((1593)"¥"Cybil, it's me. It's Harry!¥" The bullet grazes my shoulder, and with a discordant metallic sound it hits one of the wooden horses. ...She is coming towards me."); // 0x7294: 0000100c
+ShowText((1593)"¥"Cybil, W1it's me. W1It's Harry!¥"¥n¥nW2The bullet grazes my shoulder, W1and with a discordant metallic sound it hits one of the wooden horses.¥n¥nW2…She is coming towards me."); // 0x7294: 0000100c
 HandleInput(); // 0x729a: 00000010
-ShowText((1594)"Her eyes no longer appear to be human. The eerie red color of her eyes does not make it look like her life-force has been drained, but instead, it makes it appear as if she has been possessed by some evil being."); // 0x729e: 0000100c
+ShowText((1594)"Her eyes no longer appear to be human.¥nW2The eerie red color of her eyes does not make it look like her life-force has been drained, W1but instead, W1it makes it appear as if she has been possessed by some evil being."); // 0x729e: 0000100c
 HandleInput(); // 0x72a4: 00000010
 CleanScreen(); // 0x72a8: 00000064
 LoadBackground(0xaf); // 0x72ac: 00001069
 LoadEffect(0x0, 0x0, 0x1); // 0x72b2: 0000306d
 DrawBackground(); // 0x72bc: 00000065
-ShowText((1595)"I was too late, I think. Her heart had been devoured by someone, no by Alessa. What is there is no longer Cybil. Just an alter-ego of Alessa that is using Cybil's body..."); // 0x72c0: 0000100c
+ShowText((1595)"I was too late, I think.¥nW2Her heart had been invaded by someone, W1no by Alessa.¥nW2What is there is no longer Cybil.¥nW2Just an alter-ego of Alessa that is using Cybil's body…"); // 0x72c0: 0000100c
```

Comment:
Changed "devoured" to "invaded" since that is the common translation for shinshoku used in the PS1 game.